### PR TITLE
RPD magazine

### PR DESCRIPTION
**Pull request by Merek2**
## Pre-Merge Checklist
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog
:cl:
add: Added new things
add: Added more things
del: Removed old things
tweak: tweaked a few things
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

### CODE REVIEW

**Review by Tk420634 — APPROVED** [review]
I refuse to clean your litter box, now what?

File: code/modules/research/designs/ammolathe_designs.dm [hunk 1]
```diff
@@ -708,6 +708,13 @@
 	build_path = /obj/item/ammo_box/magazine/m308/ext/empty
 	category = list("initial", "Advanced Magazines")
 
+/datum/design/ammolathe/rpd
+	name = "empty drum magazine (.308)"
+	id = "rpd"
```
Comment (DameonOwen) [inline code/modules/research/designs/ammolathe_designs.dm:713]: Change this to rpdmag

Comment (TalkingCactus) [inline code/modules/research/designs/ammolathe_designs.dm:713]: id is already in use, this needs to be something else to avoid a runtime error

**Comment by Tk420634** [comment]
kill him boys

File: code/modules/research/designs/ammolathe_designs.dm [hunk 2]
```diff
@@ -708,6 +708,13 @@
 	build_path = /obj/item/ammo_box/magazine/m308/ext/empty
 	category = list("initial", "Advanced Magazines")
 
+/datum/design/ammolathe/rpd
```
Comment (TalkingCactus) [inline code/modules/research/designs/ammolathe_designs.dm:711]: also this

**Pull request merged**